MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+
+N> When tap a swiped item, the [SelectionChanging](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_SelectionChanging) and [SelectionChanged](https://help.syncfusion.com/cr/xamarin/Syncfusion.ListView.XForms.SfListView.html#Syncfusion_ListView_XForms_SfListView_SelectionChanged) events will not occur since the swiped item is reset at this time.
+
+N> When the `Visual` is `Material`, you need to set the `BackgroundColor` to `ItemTemplate` to show the slide swiping.
+
+## Assigning Start and End swipe templates
+
+The User Interface (UI) for swiping can be customized by using swipe templates `SfListView.StartSwipeTemplate` when swiping towards End and `SfListView.EndSwipeTemplate` when swiping towards Start. The contents inside the swipe template are arranged based on the offset values when swiping an item. You can reset the swiping item or swiped item by calling the `SfListView.ResetSwipeItem` method.
+
+{% tabs %}
+{% highlight xaml tabtitel=MainPage.xaml" %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+  <syncfusion:SfListView x:Name="listView" AllowSwiping="True">
+    <syncfusion:SfListView.StartSwipeTemplate>
+      <DataTemplate x:Name="StartSwipeTemplate">
+        <Grid>
+          <Grid BackgroundColor="#009EDA" HorizontalOptions="Fill" VerticalOptions="Fill" Grid.Column="0">
+            <Grid VerticalOptions="Center" HorizontalOptions="Center">
+              <Image Grid.Column="0"
+                    Grid.Row="0"
+                    BackgroundColor="Transparent"
+                    HeightRequest="35"
+                    WidthRequest="35"
+                    Source="Favorites.png" />
+            </Grid>
+          </Grid>
+        </Grid>
+      </DataTemplate>
+    </syncfusion:SfListView.StartSwipeTemplate>
+  </syncfusion:SfListView>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+//Defining start swipe template
+listView.StartSwipeTemplate = new DataTemplate(() =>
+{
+    var grid = new Grid();
+
+    var grid1 = new Grid() { BackgroundColor = Colors.FromHex("#009EDA"), HorizontalOptions = LayoutOptions.Fill, 
+                             VerticalOptions = LayoutOptions.Fill };
+    var favoriteGrid = new Grid() { HorizontalOptions = LayoutOptions.Center, VerticalOptions = LayoutOptions.Center };
+    var favoriteImage = new Image() { BackgroundColor = Colors.Transparent, HeightRequest = 35, WidthRequest = 35 };
+    favoriteImage.Source = ImageSource.FromResource("Swiping.Images.Favorites.png");
+    favoriteGrid.Children.Add(favoriteImage);
+    grid1.Children.Add(favoriteGrid);
+
+    grid.Children.Add(grid1);
+
+    return grid;
+
+});
+{% endhighlight %}
+{% endtabs %}
+
+N> Similarly, the UI for swiping towards left can be customized by using the `SfListView.EndSwipeTemplate`.
```

Comment:
N> Similarly, the UI for swiping left can be customized by using the `SfListView.EndSwipeTemplate`.

Reply:
resolved